src/main.ts

```diff
@@ -1,22 +1,27 @@
 import * as core from '@actions/core'
 import * as fs from 'fs/promises'
-import {CheckVersion} from './checkVersion'
+import {CheckVersion} from './lib/checkVersion'
 import {stringToBoolean} from './util'
 
 type TypeOfCore = typeof core
+export type ManagerType = 'npm' | 'cargo' 
 
 const ghToken: string = core.getInput('token')
-const location: string = core.getInput('npm_package_location')
-const failOnSameVersion = core.getInput('fail_on_same_version')
+const npmLocation: string = core.getInput('npm_package_location')
+const cargoLocation: string = core.getInput('cargo_package_location')
```

Comment:
these don't look like the right inputs `action.yml` has `package_manager`and `package_location`

Reply:
thanks for a flag up, has been updated.